### DISCUSSION TITLE
Rare 409 Conflict error while installing dependencies

### DISCUSSION THREAD

**Discussion by targos** [npm/feedback#673] [Registry] [answered]
It doesn't happen often, but it's definitely not the first time I see it.

See https://github.com/cheminfo/nmrium/runs/6157880389?check_suite_focus=true, https://github.com/cheminfo/nmrium/runs/6157880304?check_suite_focus=true

**Comment by monishcm** [comment]
Our support team might be able to help you here, request you to please open a support ticket.

**Reply by ljharb** [reply]
I’ve gotten this a few times over the last few days too.

It’s not really a case for individual support imo, it’s more of a transient registry error.

**Reply by arcanis** [reply]
Indeed, just seen it pop up into our E2E tests as well.

**Reply by targos** [reply]
@ljharb that's a private repo

**Reply by ljharb** [reply]
whoops, yes, i realized that but the UI didn't tell me it had posted. Deleted the comment.

**Reply by ljharb** [reply]
it happened again on a public repo: https://github.com/inspect-js/is-well-known-symbol/runs/6163765056?check_suite_focus=true

**Reply by weeyum** [reply]
also seeing this happening in a private repo of mine, it started friday afternoon PT and continues today.

**Reply by ljharb** [reply]
Here's some more:
 - https://github.com/ljharb/travis-ci/runs/6166268651?check_suite_focus=true
 - https://github.com/import-js/eslint-plugin-import/runs/6166452230?check_suite_focus=true

**Reply by ljharb** [reply]
and another that's a cloudflare 5xx: https://github.com/substack/tape/runs/6167040141?check_suite_focus=true

**Reply by monishcm** [reply]
This is being looked upon right now and should be soon resolved.

**Reply by aswan** [reply]
@monishcm I don't see anything about this on https://status.npmjs.org/
This issue is affecting us but it took a while to stumble onto this discussion, would you consider providing information on this issue through the npm status page?

**Reply by monishcm** [reply]
This is a transient error that we are seeing with our CDN. Our logs show things are working fine since this morning. We continue to monitor the situation and working with our vendor for more info into the issue.

**Reply by ljharb** [reply]
Thanks! Your cdn is still part of the npm service, from your user’s perspective, so it would still be great to include that on status.npmjs.org in the future :-)

**Reply by monishcm** [reply]
https://status.npmjs.org/incidents/ljzb0hdg8zr3

**Comment by monishcm** [comment, marked as answer]
This is a transient error that we are seeing with our CDN. Our logs show things are working fine since yesterday morning. We continue to monitor the situation and are working with our vendor for more information into this issue. The feedback to include these services as part of status.npmjs.org is being investigated.
